lib/src/state_machine.dart

```diff
@@ -99,6 +101,24 @@ class StateMachine<StateIdentifier> {
       )
     ]);
   }
+
+  /// Generate a FSM state diagram [_MermaidStateDiagram].
```

Comment:
Ya, I think we can do that. Because even though it is a private element, people still can read the function for it right?

From effective dart docs:
> CONSIDER writing doc comments for private APIs.
Doc comments aren’t just for external consumers of your library’s public API. They can also be helpful for understanding private members that are called from other parts of the library.

Reply:
Yes you are right. Let's take care of the quality of documentation for developers too.